pkg/settings/settings.go

```diff
@@ -57,6 +57,7 @@ var (
 	HarvesterCSICCMVersion = NewSetting(HarvesterCSICCMSettingName, `{"harvester-cloud-provider":">=0.0.1 <0.3.0","harvester-csi-provider":">=0.0.1 <0.3.0"}`)
 	NTPServers             = NewSetting(NTPServersSettingName, "")
 	WhiteListedSettings    = []string{"server-version", "default-storage-class", "harvester-csi-ccm-versions", "default-vm-termination-grace-period-seconds"}
+	ImagePreloadStrategy   = NewSetting(ImagePreloadStrategySettingName, SequentialImagePreload) // options are skip, sequential, and parallel
```

Comment:
We can extend upgrade CRD to control the behavior of individual upgrades. But a setting is also OK since there will be only a running upgrade. Can you refactor the setting name to upgrade-config or something and make the value a json structure, it's easier to extend upgrade-related configs without adding more settings.